### PR TITLE
Allow pretending for DDL statements

### PR DESCRIPTION
**Pull request by matthewjumpsoffbuildings**
If the Connection is pretending, the DDL statements should not run

Since affectingStatement already handles pretending, its fine to just pass through to that method which will return 0 as normal, allowing the generated SQL to be captured

### CODE REVIEW

File: src/Schema/Builder.php [hunk 1]
```diff
@@ -165,6 +166,8 @@ public function dropAllTables()
             $blueprint->drop();
             array_push($queries, ...$blueprint->toSql($connection, $this->grammar));
         }
-        $connection->runDdlBatch($queries);
+
+        if(count($queries))
+            $connection->runDdlBatch($queries);
```
Comment (taka-oyama) [inline src/Schema/Builder.php:171]: These fixes don't belong in this PR.

File: src/Connection.php [hunk 2]
```diff
@@ -295,6 +295,10 @@ public function statement($query, $bindings = []): bool
         }
 
         // is DDL Query
+        // allow pretending for DDL queries
+        if($this->pretending())
+            return $this->affectingStatement($query, $bindings) !== null;
```
Comment (taka-oyama) [inline src/Connection.php:300]: I think runDdlBatch should be fixed to handle the pretending instead of forwarding it to affectingStatement.

File: src/Schema/Builder.php (hunk 1 above)
Comment (matthewjumpsoffbuildings) [reply src/Schema/Builder.php:171]: ye fair i merged from the other drop-all branch into this

File: src/Connection.php (hunk 2 above)
Comment (matthewjumpsoffbuildings) [reply src/Connection.php:300]: but isnt affectingStatement()s implementation of `pretend` is basically complete and simple and i would just end up duplicating affectingStatement() line for line, except for this range of lines

https://github.com/colopl/laravel-spanner/blob/master/src/Connection.php#L313-L323

Comment (matthewjumpsoffbuildings) [reply src/Connection.php:300]: or is nothing in the affectingStatement method relevant?

**Comment by taka-oyama** [comment]
Fixed in https://github.com/colopl/laravel-spanner/pull/170.
